src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import cn from 'classnames';
+import React, { useState } from 'react';
+
+import { Person } from '../../types/Person';
+
+interface Props {
+  people: Person[];
+  delay?: number;
+  selected?: Person | null;
+  onSelected: (person: Person) => void;
+  query: string;
+  onQueryChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const InnerDropdown: React.FC<Props> = ({
+  people,
+  delay = 150,
```

Comment:
will be good write optional params in the end of the list

Reply:
fixed